### DISCUSSION TITLE
IO Expander for XIAO | Seeed Studio Wiki

### DISCUSSION THREAD

**Discussion by giscus** [Seeed-Studio/wiki-documents#654] [Q&A]
# IO Expander for XIAO | Seeed Studio Wiki

Create a doc page with rich content.

https://wiki.seeedstudio.com/io_expander_for_xiao/

**Comment by streetdriver02** [comment]
Will this expansion board be available in near future? It disappear from the shop. Thanks!

**Comment by alooox** [comment]
When will this be in stock again. Shop has 404 Error

**Comment by MatthewJeffson** [comment]
Hello @streetdriver02 @thealexproctor 
Thank you for your interests for the product. Unfortunately, we won’t be selling this product anymore.
Best Regards,
Matthew

**Reply by UltimateMilooow** [reply]
Hello @MatthewJeffson

Is this product still available? I sent an email and haven't gotten a response back yet, let me know if there is anything I can do!

Best regards,
Milo

**Reply by nfs0619** [reply]
Hi Milo,

Thanks for reaching out. We won’t be selling this product anymore.
